mock-backend/src/routes/productRoutes.js

```diff
@@ -1,13 +1,15 @@
 const express = require("express");
 const {
   getAllProducts,
+  getProductById,
   getAllManagerProducts,
-  createProduct
+  createProduct,
```

Comment:
maybe to create issue to configure `prettier` to avoid this `,` everywhere for mock backend ?

Reply:
I guess we can create a low-priority issue for it. In my point of view, it is not so important for now because we use mock backend just for testing

Reply:
> I guess we can create a low-priority issue for it. In my point of view, it is not so important for now because we use mock backend just for testing. Let's keep this comma, because the developer who will work on this issue will execute prettier format command and all files would be formatted at once using `.prettierrc`

Sure. ok